taxi/forms.py

```diff
@@ -0,0 +1,53 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Car, Driver
+
+
+class LicenseNumberValidationMixin:
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if (
+            len(license_number) != 8
+            or not license_number[:3].isalpha()
+            or not license_number[:3].isupper()
+            or not license_number[3:].isdigit()
+        ):
+            raise ValidationError(
+                "Ensure that value is consist only of 8 characters: "
+                "first 3 characters are uppercase letters, "
+                "last 5 characters are digits"
```

Comment:
The validation error message states that the license number should consist of '8 characters: first 3 characters are uppercase letters, last 5 characters are digits'. However, the logic checks for 8 characters in total, with the first 3 being letters and the last 5 being digits. The message should be corrected to reflect that the last 5 characters are digits, not just any characters.